### ISSUE TITLE
Crash when resizing splitter window too small

### ISSUE THREAD

**Issue by miv-ableton** [AbletonAG/aqt-cassowary#6]
While running the example Splitters.qml app, if you quickly resize the window smaller such that minimum sizes are not possible to maintain, you get a crash:

```
libc++abi.dylib: terminating with uncaught exception of type rhea::internal_error: dual_optimize: no pivot found

Process:               qmlscene [38401]
Path:                  /Volumes/VOLUME/*/qmlscene
Identifier:            qmlscene
Version:               ???
Code Type:             X86-64 (Native)
Parent Process:        Python [38398]
Responsible:           Terminal [315]
User ID:               502

Date/Time:             2015-01-28 13:49:26.833 +0100
OS Version:            Mac OS X 10.10.1 (14B25)
Report Version:        11
Anonymous UUID:        EFB97EAA-C3CD-BD5F-AD8C-26F1610BA6F4

Sleep/Wake UUID:       0FC0CBDB-F97D-4047-A59B-E860D7279480

Time Awake Since Boot: 760000 seconds
Time Since Wake:       20000 seconds

Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_CRASH (SIGABRT)
Exception Codes:       0x0000000000000000, 0x0000000000000000

Application Specific Information:
terminating with uncaught exception of type rhea::internal_error: dual_optimize: no pivot found
abort() called

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   libsystem_kernel.dylib          0x00007fff953e6282 __pthread_kill + 10
1   libsystem_c.dylib               0x00007fff96a75b73 abort + 129
2   libc++abi.dylib                 0x00007fff8be3ea21 abort_message + 257
3   libc++abi.dylib                 0x00007fff8be669b9 default_terminate_handler() + 243
4   libobjc.A.dylib                 0x00007fff90ffc6db _objc_terminate() + 124
5   libc++abi.dylib                 0x00007fff8be640a1 std::__terminate(void (*)()) + 8
6   libc++abi.dylib                 0x00007fff8be63d48 __cxa_rethrow + 99
7   libobjc.A.dylib                 0x00007fff90ffc4e4 objc_exception_rethrow + 40
8   com.apple.AppKit                0x00007fff924bc7f0 -[NSWindow(NSWindowResizing) _resizeWithEvent:] + 1778
9   com.apple.AppKit                0x00007fff91e73eb9 -[NSTitledFrame mouseDown:] + 200
10  com.apple.AppKit                0x00007fff91e73caa -[NSThemeFrame mouseDown:] + 68
11  com.apple.AppKit                0x00007fff9230cf4f -[NSWindow _reallySendEvent:] + 12827
12  com.apple.AppKit                0x00007fff91d9750c -[NSWindow sendEvent:] + 368
13  libqcocoa.dylib                 0x000000010d214717 -[QNSWindow superSendEvent:] + 39
14  libqcocoa.dylib                 0x000000010d2140bb -[QNSWindowHelper handleWindowEvent:] + 411
15  com.apple.AppKit                0x00007fff91d49096 -[NSApplication sendEvent:] + 2238
16  libqcocoa.dylib                 0x000000010d22b00e -[QNSApplication sendEvent:] + 78
17  com.apple.AppKit                0x00007fff91bd5e98 -[NSApplication run] + 711
18  libqcocoa.dylib                 0x000000010d22792d QCocoaEventDispatcher::processEvents(QFlags<QEventLoop::ProcessEventsFlag>) + 2189
19  org.qt-project.QtCore           0x000000010a0c33fd QEventLoop::exec(QFlags<QEventLoop::ProcessEventsFlag>) + 381
20  org.qt-project.QtCore           0x000000010a0c63da QCoreApplication::exec() + 346
21  qmlscene                        0x000000010947e2ec main + 11516
22  qmlscene                        0x000000010947b444 start + 52

```

**Comment by ghost** [comment]
Thanks Mike!

**Comment by ghost** [comment]
Fixed by #8
